### PR TITLE
Added climate.modbus module documentation

### PR DESCRIPTION
**Pull request by Kirchoff**
**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#12224
## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

### CODE REVIEW

**Comment by dgomes** [comment]
Can you provide more information about using this component ? E.g. What equipments works with this ? Does it require any extra hardware ?

File: source/_components/climate.modbus.markdown [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    slave: 1
+    # Control Loop 1 Setpoint
+    target_temp_register: 2782
+    # Universal Input 1 Module 1
+    current_temp_register: 27586
+    data_type: float
+
+```
+
+Configuration variables:
```
Comment (frenck) [inline source/_components/climate.modbus.markdown]: Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

File: source/_components/climate.modbus.markdown [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+climate:
+  - platform: modbus
+    name: Watlow F4T
+    # Modbus TCP
+    slave: 1
+    # Control Loop 1 Setpoint
+    target_temp_register: 2782
+    # Universal Input 1 Module 1
+    current_temp_register: 27586
+    data_type: float
```
Comment (frenck) [inline source/_components/climate.modbus.markdown]: Please don't add optional configuration parameters to the examples.
We'd like to keep examples as minimal as possible.

File: source/_components/climate.modbus.markdown [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+title: "Modbus"
+description: "Instructions how to integrate a Modbus thermostat within Home Assistant."
+date: 2018-01-29 9:35
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: modbus.png
+ha_category: Thermostat
+ha_release: pre 0.7
```
Comment (frenck) [inline source/_components/climate.modbus.markdown]: This is a new component. Please set the `ha_release` to the next release (`0.66`).

**Review by frenck — CHANGES_REQUESTED** [review]
@Kirchoff Thank you for this. Left a couple of small RFC's. Could you please take a look? 👍

**Comment by Kirchoff** [comment]
@frenck - I made the changes you requested, does everything look good? Thank you.

File: source/_components/climate.modbus.markdown [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  - platform: modbus
+    name: Watlow F4T
+    slave: 1
+    target_temp_register: 2782
+    current_temp_register: 27586
+
+```
+
+{% configuration %}
+  - **name** (*Required*): Name of the sensor.
```
Comment (frenck) [inline source/_components/climate.modbus.markdown]: Almost correct! You need to change/format the configuration options as well.

https://home-assistant.io/developers/documentation/create_page/#configuration

**Review by frenck — CHANGES_REQUESTED** [review]
@Kirchoff Almost good!

**Comment by frenck** [comment]
Rebased the PR in order to fix the build.

**Review by frenck — DISMISSED** [review]
Thank you @Kirchoff! 🏅 

Can be merged as soon as the parent PR gets merged.

**Review by MartinHjelmare — APPROVED** [review]

**Pull request merged**